ghost/core/test/unit/server/web/api/middleware/cors.test.js

```
@@ -1,3 +1,4 @@
+const assert = require('node:assert/strict');
```

ESLint

[error] 'assert' is assigned a value but never used. Allowed unused vars must match /^should$/u.

(no-unused-vars)